t/t2611-debug-emulate.t

```diff
@@ -13,7 +13,7 @@ stop_tasks_test()     {
 
 test_under_flux 2
 
-TIMEOUT=10
+TIMEOUT=100
```

Comment:
The test at the end expects to fail with a timeout, so we don't want to blindly increase the timeout 10x in this test. (Did this actually succeed for you?)

The last test has:
```sh
        test_must_fail flux job wait-event -vt ${TIMEOUT} ${jobid} finish &&
```
In order to check that the job does _not_ finish after attach by `flux job attach --debug`. If you increase the timeout to 100 then the test will take a minimum of 100 seconds to run... and other timeouts may come into play ...

Reply:
hmmm, well it's failing for me now.  I have no idea how this passed for me before.  I clearly did something stupid before.  But yeah, lets update that one line to use a timeout of 2.